src/inspector/value-mirror.cc

```diff
@@ -1388,7 +1388,13 @@ bool ValueMirror::getProperties(v8::Local<v8::Context> context,
     CHECK(tryCatch.HasCaught());
     return false;
   }
+
+  int i = 0;
   while (!iterator->Done()) {
+    if (v8::recordreplay::HasDivergedFromRecording() && ++i > 1000) {
+      // Quick-and-dirty fix for slow Pause object queries + getAllFrames (RUN-1315)
+      break;
```

Comment:
Having a recordreplay::Print statement here might be nice so we can more clearly indicate the overflow occurred.

Reply:
The one time where I remove the `Print` statement because I thought @bhackett1024 might not want me to add it 🤣 

I'm adding it back 😊